lib/ecto/adapters/postgres.ex

```diff
@@ -367,8 +376,10 @@ defmodule Ecto.Adapters.Postgres do
     {:ok, path}
   end
 
-  defp append_versions(table, versions, path) do
-    sql = Enum.map_join(versions, &~s[INSERT INTO public."#{table}" (version) VALUES (#{&1});\n])
+  defp append_versions(table, prefix_versions, path) do
+    sql = Enum.map_join(prefix_versions, fn {prefix, version} ->
+      ~s[INSERT INTO #{prefix}."#{table}" (version) VALUES (#{version});\n]
+    end)
```

Comment:
```suggestion
    sql =
      Enum.map_join(versions, fn {prefix, prefix_versions} ->
        Enum.map_join(
          prefix_versions,
          &~s[INSERT INTO #{prefix}."#{table}" (version) VALUES (#{&1});\n]
        )
      end)
```

Formatting and also has to be nested map join because each prefix has its own list of versions.